### DISCUSSION TITLE
u8g2 I2C with RISCV CH32V305/7 MCU

### DISCUSSION THREAD

**Discussion by M-Minhaj** [olikraus/u8g2#1963] [Show and tell]
if anyone wants to use u8g2 with RISCV CH32V305/7 microcontrollers.

[here](https://github.com/M-Minhaj/u8g2-with-RISC-V-MCU---CH32V305-7-MCU/blob/main/WCH32_OLED_I2C/User/main.c)

Thank you

**Comment by olikraus** [comment]
Great, I have linked your project from the porting page: https://github.com/olikraus/u8g2/wiki/Porting-to-new-MCU-platform#riscv

**Comment by linhz0hz** [comment]
Thank you! Exactly what I was looking for.
